pkg/alertmanager/alertmanager.go

```diff
@@ -81,6 +81,21 @@ func (s Service) Configure(ctx context.Context, secret *v1.Secret) error {
 	}
 
 	logger.Info("Alertmanager: configured")
+
+	// TODO Clean up the anonymous tenant alertmanager deletion code in the next release
+	logger.Info("Alertmanager: cleaning up anonymous tenant")
```

Comment:
The goal of this PR is not to produce pretty code because I plan to remove it once this is released as this is a one time clean up